index.html

```diff
@@ -190,30 +190,38 @@ <h3>A Special Time</h3>
       <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
       <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
       <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
-      <form action="">
+
+      <h3 class="line-bottom">LEAVE A COMMENT</h3>
+      <form action="#" method="POST" class="form">
         <label for="user-name">Name</label>
-        <input type="text" name="user-name" id="user-name" maxlength="20" required>
+        <input type="text" name="user-name" id="user-name" maxlength="20" placeholder="Full Name" required>
 
         <label for="user-email">Email</label>
-        <input type="email" name="user-email" id="user-email" maxlength="20" required>
+        <input type="email" name="user-email" id="user-email" maxlength="20" placeholder="Email Address" required>
 
         <label for="text-field">Comment</label>
         <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Type your comment here…" maxlength="250" required></textarea>
```

Comment:
Hello, 
The form looks awesome except for some two adjustments below.
Make the text fields stretch the entire column width. Or add both name and email text fields on the same line to fit the entire column width. Or move the whole form element unto the center of the column.
Also make radio buttons to select only one option at a time